xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+      Find the card vendor and model IDs. Utilize the bus number identified in
+      <xref linkend="gpu-passthru-verify-host"/>, for example
+      <literal>03:00.0</literal>:
+     </para>
+<screen>
+&prompt.user;lspci -nn | grep 03:00.0
+03:00.0 3D controller [0302]: NVIDIA Corporation GV100 [Tesla V100 PCIe] [10de:1db4] (rev a1)
+</screen>
+    </step>
+    <step>
+     <para>
+      Create the file <filename>vfio.conf</filename> in the
+      <filename>/etc/modprobe.d/</filename> directory with the following
+      content:
+     </para>
+<screen>options vfio-pci ids=10de:1db4</screen>
+     <note>
+      <para>
+       Double check that your card does not need an extra <option>ids=</option>
+       parameter. For example, for Public Nvidia Card you need to also add the
+       audio device in the list, otherwise you will not be able to use the
+       card.
+      </para>
+     </note>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-load-vfio">
+   <title>Load the <xref linkend="gloss-vt-acronym-vfio"/> driver</title>
+   <para>
+    There are three ways you can load the
+    <xref linkend="gloss-vt-acronym-vfio"/> driver.
+   </para>
+   <sect3 xml:id="gpu-passthru-load-vfio-initrd">
+    <title>Including the driver in the initrd file</title>
+    <procedure>
+     <step>
+      <para>
+       Create the file
+       <filename>/etc/dracut.conf.d/gpu-passthrough.conf</filename> and add the
+       following content:
+      </para>
+<screen>add_drivers+="vfio vfio_iommu_type1 vfio_pci vfio_virqfd"</screen>
+     </step>
+     <step>
+      <para>
+       Re-generate the initrd file:
+      </para>
+<screen>&prompt.sudo;dracut --force /boot/initrd $(uname -r)</screen>
+     </step>
+    </procedure>
+   </sect3>
+   <sect3 xml:id="gpu-passthru-load-vfio-modules">
+    <title>Adding the driver to the list of auto-loaded modules</title>
+    <para>
+     Create the file <filename>/etc/modules-load.d/vfio-pci.conf</filename> and
+     add the following content:
+    </para>
+<screen>
+pci_stub
+vfio
+vfio_iommu_type1
+vfio_pci
+kvm
+kvm_intel
+</screen>
+   </sect3>
+   <sect3 xml:id="gpu-passthru-load-vfio-manual">
+    <title>Loading the driver manually</title>
+    <para>
+     To load the driver manually at run-time, execute the following command:
+    </para>
+<screen>&prompt.sudo;modprobe vfio-pci</screen>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-disable-msr">
+   <title>Disable MSR for &mswin; guests</title>
+   <para>
+    For &mswin; guests, we recommend disabling MSR (model-specific register) to
+    avoid crash of the guest. Create the file
+    <filename>/etc/modprobe.d/kvm.conf</filename> and add the following
+    content:
+   </para>
+<screen>options kvm ignore_msrs=1</screen>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-ovmf">
+   <title>Install and enable UEFI firmware</title>
+   <para>
+    For proper &gpuback; functionality, the host need to boot using the UEFI
+    firmware.
```

Comment:
```suggestion
    firmware (that is, not using a legacy-style BIOS boot sequence).
```

Reply:
This suggestion is partly to reduce ambiguity -- it is not possible to choose the type of firmware in a computer, but it is possible to choose what boot mode the OS uses.